site/content/docs/5.1/components/footer.md

```diff
@@ -0,0 +1,131 @@
+---
+layout: docs
+title: Footer
+description: Documentation and examples for a powerful and responsive Boosted's footer. It includes support for branding, navigation and more.
+group: components
+aliases: "/docs/5.1/components/footer/"
+toc: true
+---
+
+## How it works
+
+The footer is basically a `<footer>` tag with a `.bg-dark` class. It comes with 5 ready to use bars which should be arranged vertically in the following order:
+- Title with content bar
+- Follow us bar
+- Navigation bar
+- Service bar
+- Terms & Conditions **mandatory** bar
+
+You can choose to display each part or not, except the last one which is mandatory. No matter how many parts you use, they should follow the order listed above.
+
+If these available parts don't meet your specific needs, feel free to develop your own custom part accordingly to the Orange Design System.
+
+## Example
+
+{{< orange-footer mailing=true follow=true nav=true service=true mandatory=true id="1" >}}
+{{< /orange-footer >}}
+
+## Footer bars
+
+{{< callout info >}}
+The `<h2></h2>` and `<h3></h3>` tags used here are informative, you **need** to adapt those to your website's title hierarchy.
+{{< /callout >}}
+
+The footer bars are independent from each other. Just add a dark separator between each bar of the final footer.
+
+```html
+<div class="border-bottom border-1 border-dark"></div>
+```
+
+### Title with content bar
+
+This footer bar is a container with a `.footer-title-content` class. This container includes a title and a `.row` container in which to add your content. See below an example with a mail form content.
+
+{{< example class="p-0" >}}
+{{< orange-footer mailing=true >}}
+{{< /orange-footer >}}
+{{< /example >}}
+
+### Follow us bar
+
+This footer bar is a container with `.footer-social` class into which a list of social media icons can be added:
+
+<div class="bd-example p-0">
+  {{< orange-footer follow=true >}}
+  {{< /orange-footer >}}
+</div>
+
+```html
+<div class="container-xxl footer-social">
```

Comment:
Since we try to use the shortcode as much as possible, I think that the `<footer>` should be in the code even if the copy paste is a bit harder. Implemented like this in next commit.

Reply:
Weird rendering with the indentation

![Screenshot from 2022-04-22 11-19-06](https://user-images.githubusercontent.com/17381666/164676757-e6621480-cb1f-4e13-955e-a8cc3bdfcd2e.png)